### DISCUSSION TITLE
Regain access to organisation

### DISCUSSION THREAD

**Discussion by EddieDemon** [community/community#88697] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,

On my company account I have tried to create a new organisation with my company name, this one however has been registered by someone else, I suspect an old employee. How can I gain access to said organisation?

**Comment by Andrkenya** [comment]
Hi Eddie,

In this scenario, two users can't register a similar company name, please check out the new owner who registered that name and discuss. Or else just slightly change the name and create a new org.

**Reply by EddieDemon** [reply]
Hi there,

Like I said, it's the account of an old colleague. He has left the company and the account is registered with an address of the company

**Comment by EddieDemon** [comment, marked as answer]
We gained access to the old colleagues account and took it over.

**Reply by joellyons** [reply]
@EddieDemon  How did you manage that? Were you able to contact the former employee and get their help or is there another way? We're in the same predicament but can't seem to contact the former employee.

**Reply by EddieDemon** [reply]
I may or may not have requested an administrator to attach their email address as alias to my mailbox. Luckly there was not any MFA enabled. That might be your blockade.

**Comment by joellyons** [comment]
Ah, good idea! Thanks 😊On Jul 2, 2024 12:31 PM, EddieDemon ***@***.***> wrote:
I may or may not have requested an administrator to attach their email address as alias to my mailbox. Luckly there was not any MFA enabled. That might be your blockade.
